resource-watch/resource-watch
# Updates header component

## Overview
- Aligns user icon in the app header (special dedication to @mbarrenechea):
![image](https://user-images.githubusercontent.com/999124/54834825-41550200-4cc1-11e9-9105-8fc0cd9c52ce.png)

- Fixes styling bug with search bar in Firefox 66.
https://files.slack.com/files-pri/T052A927W-FGYCQADMX/image.png

- Moves app and admin header items to constants instead of keeping them in the store for no reason 🤔.
- Split search into two components: `search-bar` and `search-results`. Before it was all merged in one folder.

- Moves some styles from global scope to component-level.
-Renames some files with old naming to current one.
- Some eslint here and there.

## Testing instructions
Check app header in any page (also `/admin` one). For the search, go to `/search` and you can test both search at the same time.


## Pivotal task
Involved tasks: https://www.pivotaltracker.com/story/show/164700670

---

## Checklist before submitting
